code/_onclick/hud/living.dm

```diff
@@ -11,6 +11,12 @@
 	pull_icon.hud = src
 	static_inventory += pull_icon
 
-	healths = new /obj/screen/healths/living()
-	healths.hud = src
-	infodisplay += healths
+
+	//explanation: since living mobs have no limbs, they only need one of these to show health nicely.
+	//while health doll is cooler, it does not work on huge mobs and so they use the basic version.
+	if(owner.mob_biotypes & MOB_EPIC)
```

Comment:
perhaps a better way to do this, may revisit

Reply:
You can just check icon size directly.

Reply:
One of those medical + signs would be good as a fallback

Reply:
kk.

Reply:
You could probably make these single type or subtype.